packages/back-office-service/prisma/migrations/20230316101542_init/migration.sql

```diff
@@ -0,0 +1,74 @@
+BEGIN TRY
+
+BEGIN TRAN;
+
+-- CreateTable
+CREATE TABLE [dbo].[Project] (
+    [id] INT NOT NULL IDENTITY(1,1),
+    [caseId] INT NOT NULL,
+    [caseReference] NVARCHAR(1000),
+    [projectName] NVARCHAR(1000),
+    [projectDescription] NVARCHAR(1000),
+    [publishStatus] NVARCHAR(1000),
+    [sector] NVARCHAR(1000),
+    [projectType] NVARCHAR(1000),
+    [sourceSystem] NVARCHAR(1000) NOT NULL,
+    [stage] NVARCHAR(1000),
+    [projectLocation] NVARCHAR(1000),
+    [projectEmailAddress] NVARCHAR(1000),
+    [regions] NVARCHAR(1000),
+    [transboundary] BIT,
+    [easting] INT,
+    [northing] INT,
+    [welshLanguage] BIT,
+    [mapZoomLevel] NVARCHAR(1000),
+    [secretaryOfState] NVARCHAR(1000),
+    [dateProjectAppearsOnWebsite] NVARCHAR(1000),
+    [dateOfDCOAcceptance] NVARCHAR(1000),
+    [anticipatedDateOfSubmission] NVARCHAR(1000),
+    [anticipatedSubmissionDateNonSpecific] NVARCHAR(1000),
+    [dateOfDCOSubmission] NVARCHAR(1000),
```

Comment:
NVARCHAR(1000) for a date?

Reply:
Yeah, that is excessive.. I'll have a look at defining lengths in the Prisma schema file

Reply:
We are getting them from Back Office as Strings, so was going to store as we get them. Could convert to a date type though if that would be better?

Reply:
I would say so - assuming they are ISO 8601 etc, would be better for our DB to have date types